docs/conf.py

```diff
@@ -21,7 +21,7 @@
 
 # Mock imports (things that can't be installed at do building time)
 
-autodoc_mock_imports = ["numpy", "pandas", "matplotlib", "scipy"]
+autodoc_mock_imports = ["numpy", "pandas", "pyyaml"]
```

Comment:
The mocked imports are used by Sphinx when building the documentation, without them you get a bunch of error messages about not being able to import things. This may no longer be the case but was why those were added.

To check build the documents locally from the repository with...

```bash
pip install -e .[docs]
cd docs
sphinx-multiversion . _build/html
```

We use Sphinx-multiversion so that it builds documentation for each tagged version of the code that has been released. If there are no complaints about `matplotlib` or `scipy` and adding `pyyaml` suppresses this you're good to go.

Reply:
It definitely returns some warnings, but nothing about the removed dependencies and it's the same output as the current master branch :)

That's good to know though! I'll probably need to help updating some of those nice docs soon!